### ISSUE TITLE
validation loss and validation accuracy doesnot compliment each other

### ISSUE THREAD

**Issue by sameerkhann** [NVIDIA/DIGITS#737]
i trained my network and i recorded the validation loss 0.06 and validation accuracy as 97.7. But these numbers doesnot compliment each in the sense the should sum up to 1 or 100. the  validation loss should be 2.3 where as the loss is 0.06. can you please explain it please

**Comment by gheinrich** [comment]
@sameerkhann: the logistic loss does not measure the proportion of wrong classifications. You might want to review the formulation [there](http://caffe.berkeleyvision.org/doxygen/classcaffe_1_1SoftmaxWithLossLayer.html#details). In practice `loss+accuracy` is almost never equal to `100`.
